export const onEnd = function () {
  throw new Error('test')
}
